CMake/cdat_modules_extra/setup_runtime.sh.in

```diff
@@ -1,6 +1,18 @@
 # Everything beyond this point will be determined relatively
 # from this path.
 install_prefix="@CMAKE_INSTALL_PREFIX@"
+# Reset previous uvcdat env messing up
+source @CMAKE_INSTALL_PREFIX@/bin/reset_runtime.sh
+
+# Saves what we will mess with
+for v in PATH LD_LIBRARY_PATH DYLD_LIBRARY_PATH PYTHONPATH OPAL_PREFIX LIBOVERLAY_SCROLLBAR ; do
+   tmp="${v}"
+    if [ -n "${!tmp}" ] ; then
```

Comment:
well, i think that up to now we had a .sh and a .csh that seemed to cover everything, is zsh broken by this? Or just the reset?